internal/compiler/widgets/material-base/widget-lineedit.slint

```diff
@@ -0,0 +1,93 @@
+// Copyright © SixtyFPS GmbH <info@slint-ui.com>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-commercial
+
+
+import { md } from "md.slint";
+
+// Single line text input field with Material Design Outline TextField look and feel.
+export LineEdit := Rectangle {
+    property <length> font-size <=> input.font-size;
+    property <string> text <=> input.text;
+    property <string> placeholder-text <=> placeholder.text;
+    property <bool> has-focus: input.has-focus;
+    property <bool> enabled <=> input.enabled;
+    property input-type <=> input.input-type;
+    property horizontal-alignment <=> input.horizontal-alignment;
+    property read-only <=> input.read-only;
+    callback accepted <=> input.accepted;
+    callback edited <=> input.edited;
+   
+    min-width: 120pt;
+    height: 56pt;
```

Comment:
Should this perhaps be a `min-height`? If the `font-size` of the `LineEdit` is set to a value that exceeds this, it would look strange. Or is there a way to express this height relative to the font size?

Reply:
It is a fixed height definition. No `min-height`. In most style guides widgets like TextField and Buttons has always a fixed height. The font-size should not be set to a value that expands the widget. Therefore there is also a font-size defined in the md guidelines.

Reply:
Okay :)